src/components/Patient/PatientRegistration.tsx

```diff
@@ -179,6 +179,9 @@ export default function PatientRegistration(
       same_phone_number: false,
       same_address: true,
     },
+    mode: "all",
+    criteriaMode: "all",
+    reValidateMode: "onChange",
```

Comment:
isn't this sufficient? 🤔 

```suggestion
    mode: "onChange",
```